docs/getting-started/user-roadmap-zowe-explorer.md

```diff
@@ -1,121 +1,59 @@
-
-* [**Developing for Eclipse Theia**](https://github.com/zowe/zowe-explorer-vscode/wiki/Developing-for-Theia)
-
-   This article contains information on how to develop for Eclipse Theia.
+<iframe class="embed-responsive-item" id="youtubeplayer2" title="How to Work with Data Sets" type="text/html" width="100%" height="365" src="https://www.youtube.com/embed/X4oSHrI4oN4" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen="true"> </iframe>
 
 ## Contributing to Zowe Explorer
 
-> Zowe skill level: Advanced
-
-* [**Contributing guidelines**](https://github.com/zowe/zowe-explorer-vscode/blob/master/CONTRIBUTING.md)
-
-   This document is intended to be a living summary of conventions & best practices for development of the Visual Studio Code Extension for Zowe.
-
-* [**Conformance Program**](../extend/zowe-conformance-program.md)
-
-  This topic introduces the Zowe Conformance Program. Conformance provides Independent Software Vendors (ISVs), System Integrators (SIs), and end users greater confidence that their software will behave as expected. As vendors, you are invited to submit conformance testing results for review and approval by the Open Mainframe Project. If your company provides software based on Zowe CLI, you are encouraged to get certified today.
-
-* [**Blog: Zowe Conformance Program Explained**](https://medium.com/zowe/zowe-conformance-program-7f1574ade8ea)
-
-   This Medium article provide more details about the Conformance Program, including useful references.
-
-## Troubleshooting and support
-
-* [**Troubleshooting Zowe Explorer**](../troubleshoot/ze/troubleshoot-ze.md)
-
-   Learn about the tools and techniques that are available to help you troubleshoot and resolve problems. You can also find the list of Zowe Explorer issues.
+- Review the [**Contribution Guidelines**](https://github.com/zowe/vscode-extension-for-zowe/blob/master/CONTRIBUTING.md) for a summary of conventions and best practices for development of the Visual Studio Code Extension for Zowe.
 
-* [**Submit an issue**](https://github.com/zowe/zowe-explorer-vscode/issues)
+## Zowe Explorer community resources
 
-   If you have an issue that is specific to Zowe Explorer, you can submit an issue against the `vscode-extension-for-zowe` repository.
+The Zowe ecosystem is more than a collection of applications and extensions. An entire community exists to work on enhancements, help answer questions, and discuss plans for the future of the open source project.
 
-## Community resources
+- Join the [**#zowe-explorer-vsce**](https://app.slack.com/client/T1BAJVCTY/CUVE37Z5F) Slack channel to ask questions, propose new ideas, and interact with the Zowe community.
 
-* [**Slack channel**](https://openmainframeproject.slack.com/)
+- You can join **Zowe Explorer squad meetings** to get involved. The meeting schedule is posted in the [Zowe calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar).
```

Comment:
For the calendar we may want to point to this link instead: https://zoom-lfx.platform.linuxfoundation.org/meetings/zowe

I believe it was mentioned on a recent TSC call that they plan to remove the `zowe-dev` mailing list at some point and the new calendar will be available on LFX Platform at the above link.